darwin/datatypes.py

```diff
@@ -565,9 +566,18 @@ def make_polygon(
     Annotation
         A polygon ``Annotation``.
     """
+
+    if darwin_v1:
+        polygon_data = {"path": point_path}
+    else:
+        # Lets handle darwin V2 datasets
+        if not isinstance(point_path[0], list):
+            point_path = [point_path]
+        polygon_data = {"paths": point_path}
+
```

Comment:
Do we still want to keep support for darwin_v1 ?

Reply:
so v1 is now deprecated, as I understand it's just some latent stuff in AI that require it. If you're moving away from v1 here, then I'm happy to ignore v1 support.

Reply:
so v1 is now deprecated, as I understand it's just some latent stuff in AI that require it. If you're moving away from v1 here, then I'm happy to ignore v1 support.

Reply:
This PR is part of us moving away from darwin v1, so we won't need it anymore. But tests might need to be re-written, in particular these:

- https://github.com/v7labs/darwin-py/blob/36ca721102507048319c8ffcf9f9373137653460/darwin/exporter/formats/coco.py#L576

- https://github.com/v7labs/darwin-py/blob/36ca721102507048319c8ffcf9f9373137653460/darwin/utils/utils.py#L749

- https://github.com/v7labs/darwin-py/blob/36ca721102507048319c8ffcf9f9373137653460/tests/darwin/datatypes_test.py#L10

- https://github.com/v7labs/darwin-py/blob/36ca721102507048319c8ffcf9f9373137653460/tests/darwin/datatypes_test.py#L21

Reply:
According to slack conv, we will keep the option for darwin-v1 format for now. Team darwin-py can remove it later.

https://vseven.slack.com/archives/C016XJEPU78/p1700578652840489